core/src/main/resources/db/changelog/changesets/2023_05_15_dataset_creation.yaml

```diff
@@ -0,0 +1,12 @@
+databaseChangeLog:
+  - changeSet:
+      id: "dataset_creation"
+      author: mbemis
+      changes:
+        - dropNotNullConstraint:
+            columnName:  dataset_id
+            tableName:  dataset
+        - addColumn:
+            tableName: dataset
+            columns:
+              - column: { name: status, type: text, constraints: { nullable: false }, defaultValue: "succeeded" }
```

Comment:
Up until this point, we've only saved datasets that have succeeded, so this is a sane backfill value.

Reply:
Is this status going to be a java enum?  Should this be "SUCCEEDED"?